### PR TITLE
[docs][ybm] Federated authentication

### PR DESCRIPTION
**Pull request by ddhodge**
Add federated authentication using Azure AD IdP
CLOUDGA-17540

@netlify /preview/yugabyte-cloud/managed-security/managed-authentication/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Note that after federated authentication is enabled, only Admin users can sign in using email-based login.
+
+Currently only the Azure Active Directory (AAD) IdP and the OIDC protocol are supported.
+
+### Prerequisites
+
+Before configuring federated authentication, keep in mind the following:
+
+- Be sure to allow popups from your IdP. While configuring federated authentication, the provider needs to confirm your identity in a new window.
+- Use your own account to test the connection.
```
Comment (jagdish-yb) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:48]: Use your own AAD account?

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+#### AAD prerequisites
+
+To use AAD for your IdP, you need to register an application in the Azure portal so the Microsoft identity platform can provide authentication and authorization services for your application. The sign-in audience for the application should be **Accounts in any organizational directory** (Multitenant).
```
Comment (jagdish-yb) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:52]: May be AAD App creation screenshot with "multitenant" option selected would help here.

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+### Configure
+
+To configure federated authentication, do the following:
+
+1. Navigate to **Security > Access Control > Authentication**, then click **Enable Federated Authentication** to display the **Enable Federated Authentication** dialog.
+1. Enter your AAD application client ID and secret.
+1. Click **Enable**.
+
+You are redirected to sign in to your IdP to test the connection.
```
Comment (jagdish-yb) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:69]: You are redirected to sign in to your IdP to test the connection. Once test connection is successful, FedAuth will be enabled.

**Review by jagdish-yb — APPROVED** [review]
lgtm.

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+- Be sure to allow popups from your IdP. While configuring federated authentication, the provider needs to confirm your identity in a new window.
+- Use your own AAD account to test the connection.
+
+#### AAD prerequisites
+
+To use AAD for your IdP, you need to register an application in the Azure portal so the Microsoft identity platform can provide authentication and authorization services for your application. The sign-in audience for the application should be **Accounts in any organizational directory** (Multitenant).
+
+![Azure account types](/images/yb-cloud/managed-authentication-azure-account-types.png)
+
```
Comment (atharva-yb) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:55]: We need to provide the Redirect URI as well right @jagdish-yb ?

Comment (jagdish-yb) [reply docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:55]: Yes, thanks @atharva-yb for catching this!
Could you please share the prod okta redirect URL with @ddhodge.
May be a screenshot of full configs will help.

**Review by atharva-yb — APPROVED** [review]

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+The available social logins include Google, GitHub, and LinkedIn. All three are enabled by default.
+
+To manage the social logins available to your account users, do the following:
+
+1. Navigate to **Security > Access Control > Authentication**, then click **Edit Configuration** to display the **Login Methods** dialog.
+1. Enable the social logins you want to use.
+1. Click **Save Changes**.
+
+If you revoke a social login that is already in use, users using that social login can either [reset their password](#reset-your-password) to configure email-based login, or sign in using a different social login. The social account must be associated with the same email address.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:33]: reset-your-password is a broken link.

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+Note that after federated authentication is enabled, only Admin users can sign in using email-based login.
+
+Currently only the Azure Active Directory (AAD) IdP and the OIDC protocol are supported.
+
+### Prerequisites
+
+Before configuring federated authentication, keep in mind the following:
+
+- Be sure to allow popups from your IdP. While configuring federated authentication, the provider needs to confirm your identity in a new window.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:47]:
```suggestion
- Be sure to allow pop-ups from your IdP. While configuring federated authentication, the provider needs to confirm your identity in a new window.
```

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+### Configure
+
+To configure federated authentication, do the following:
+
+1. Navigate to **Security > Access Control > Authentication**, then click **Enable Federated Authentication** to display the **Enable Federated Authentication** dialog.
+1. Enter your AAD application client ID and secret.
+1. Click **Enable**.
+
+You are redirected to sign in to your IdP to test the connection. Once test connection is successful, federated authentication is enabled
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:78]:
```suggestion
You are redirected to sign in to your IdP to test the connection. Once test connection is successful, federated authentication is enabled.
```

**Review by aishwarya24 — COMMENTED** [review]
Minor comments. Pending - need to still look at the YBM cloud portal.

**Pull request merged**